srcpkgs/libtracefs/patches/musl.patch

```diff
@@ -0,0 +1,15 @@
+--- a/include/tracefs.h
++++ b/include/tracefs.h
+@@ -10,6 +10,12 @@
+ #include <sched.h>
+ #include <event-parse.h>
+ 
++#ifndef __GLIBC__
```

Comment:
I don't think `__GLIBC__` is correct choice, even glibc put `cpu_set_t` under `#ifdef _GNU_SOURCE`, someone on glibc may `-U_GNU_SOURCE`

Reply:
it didn't work for me on musl with the _GNU_SOURCE guard

Reply:
Which arch are you talking about? I did the change with `x86_64-musl` and `armv6l-musl`

Reply:
i don't remember, maybe it was a side effect of something else

Reply:
where is `cpu_set_t` put under `#ifdef _GNU_SOURCE`? `_GNU_SOURCE` does not exist anywhere in neither `sched.h` nor `cpu-set.h`. Because of this, I think `__GLIBC__` is the correct way

Reply:
`man 3 CPU_SET` It's documented in the next line ;) All `cpu_set_t` related information was put into this man-page.

https://man7.org/linux/man-pages/man3/CPU_COUNT.3.html#DESCRIPTION

Reply:
It's GNU Extension, GLIBC puts it under `_GNU_SOURCE`, musl follows suite.